### DISCUSSION TITLE
loadPyodide take too long

### DISCUSSION THREAD

**Discussion by Alii-isk** [pyodide/pyodide#2546] [Q&A]
loading time is about 10s until "Loading distutils' appears on the console
i am not loading any other packages i just want a ready python environment

**Comment by rth** [comment]
It's a known issue. There is active work to improve the situation see https://github.com/pyodide/pyodide/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22size+%26+load+time+optimization%22 but it takes time. What you could do now, is to load Pyodide without distutils which is part of stdlib but is deprecated,
```
let pyodide = await loadPyodide({fullStdLib: false});
```
the overall download size should be about 12% smaller.

**Reply by Alii-isk** [reply]
i added `{fullStdLib: false} ` option but still the load time almost the same
